sponsorship_switzerland/models/contracts.py

```diff
@@ -213,7 +213,10 @@ def contract_active(self):
 
     @api.multi
     def contract_waiting_mandate(self):
-        self.write({'state': 'mandate'})
+        self.write({
+            'state': 'mandate',
+            'start_date': datetime.today()
```

Comment:
This should be done in module `sponsorship_switzerland` instead.

Reply:
We are in module `sponsorship_switzerland`, no ?